packages/experimental/Avatar/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@fluentui-react-native/experimental-avatar",
+  "version": "0.1.0",
+  "description": "An Avatar view",
+  "main": "src/index.ts",
+  "module": "src/index.ts",
+  "typings": "lib/index.d.ts",
+  "onPublish": {
+    "main": "lib-commonjs/index.js",
+    "module": "lib/index.js"
+  },
+  "scripts": {
+    "build": "fluentui-scripts build",
+    "just": "fluentui-scripts",
+    "clean": "fluentui-scripts clean",
+    "lint": "fluentui-scripts eslint",
+    "test": "fluentui-scripts jest",
```

Comment:
should this also have a depcheck task?

Reply:
Just for more context, any package that has a depcheck task (generally `"depcheck": "fluentui-scripts depcheck"`) will have that task run as part of the CI loop to check for missing or unused dependency declarations.

Reply:
Must have missed it, thanks!